serverless.yml

```diff
@@ -10,12 +10,12 @@ provider:
     COGNITO_APP_CLIENT_ID: "7i01fral9t0fdtodp78hi3vqrh"
     PATH_PARAMETER_GROUP_NAME: "groupName"
     # Get all secret key details from SSM
-    KEY_PRIVATE_EXPONENT: ${ssm:custom-aws-idp-private-key-private-exponent}
-    KEY_PRIME_P: ${ssm:custom-aws-idp-private-key-prime-p}
-    KEY_PRIME_Q: ${ssm:custom-aws-idp-private-key-prime-q}
-    KEY_PRIME_EXPONENT_P: ${ssm:custom-aws-idp-private-key-prime-exponent-p}
-    KEY_PRIME_EXPONENT_Q: ${ssm:custom-aws-idp-private-key-prime-exponent-q}
-    KEY_CRT_COEFFICIENT: ${ssm:custom-aws-idp-private-key-crt-coefficient}
+    KEY_PRIVATE_EXPONENT_NAME: "custom-aws-idp-private-key-private-exponent"
+    KEY_PRIME_P_NAME: "custom-aws-idp-private-key-prime-p"
+    KEY_PRIME_Q_NAME: "custom-aws-idp-private-key-prime-q"
+    KEY_PRIME_EXPONENT_P_NAME: "custom-aws-idp-private-key-prime-exponent-p"
+    KEY_PRIME_EXPONENT_Q_NAME: "custom-aws-idp-private-key-prime-exponent-q"
+    KEY_CRT_COEFFICIENT_NAME: "custom-aws-idp-private-key-crt-coefficient"
```

Comment:
i'm noticing that these strings in `serverless.yml` are now the same as the ones in the `build.gradle`. i wonder if there's a way to sync them, if we intend for them to always be the same?

Reply:
The ones in build.gradle are there to ensure the environment is set correctly when running `gradle test`. Unfortunately I felt I had to set the environment variables for testing twice, since running tests directly in IntelliJ did not pick up these `gradle test` environment variables, and I wanted to support both uses. Too many unrelated config/build tools in one repo!